rocketchat.py

```diff
@@ -41,18 +41,16 @@
     }
 }
 
-TEMPLATE_SERVICE = "{hostalias}/{servicedesc} is {servicestate}:\n{serviceoutput}" #noqa
+TEMPLATE_SERVICE = "{hostalias} {servicedesc} is {servicestate}:\n{serviceoutput}" #noqa
 TEMPLATE_HOST = "Host {hostalias} is {hoststate}:\n{hostoutput}"  #noqa
 
-
 def parse():
     parser = argparse.ArgumentParser(description='Sends Rocket.Chat webhooks')
     parser.add_argument('--url', help='Webhook URL', required=True)
     parser.add_argument('--proxy', help='http(s) proxy')
     parser.add_argument('--channel', help='Rocket.Chat channel')
     parser.add_argument('--hostalias', help='Host Alias', required=True)
-    parser.add_argument('--notificationtype', help='Notification type',
```

Comment:
Why you are removing the notification type?

Reply:
Found the answer: wasn't used.

Reply:
thx for taking the time to look at it. Indeed it was not used.

I'm running this version in production since last July and it works really fine.